examples/design/EU-DEMO/EUDEMO_builders/blanket.py

```diff
@@ -0,0 +1,188 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2022 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+"""
+EUDEMO builder for blanket
+"""
+from typing import Dict, List, Type, Union
+
+import numpy as np
+
+from bluemira.base.builder import Builder, Component, ComponentManager
+from bluemira.base.components import PhysicalComponent
+from bluemira.base.parameter_frame import NewParameter as Parameter
+from bluemira.base.parameter_frame import parameter_frame
+from bluemira.builders.tools import (
+    circular_pattern_component,
+    get_n_sectors,
+    pattern_revolved_silhouette,
+)
+from bluemira.display.palettes import BLUE_PALETTE
+from bluemira.geometry.face import BluemiraFace
+from bluemira.geometry.placement import BluemiraPlacement
+from bluemira.geometry.tools import boolean_cut, make_polygon, slice_shape
+
+
+class Blanket(ComponentManager):
+    """
+    Wrapper around a Blanket component tree.
+    """
+
+
+@parameter_frame
+class BlanketBuilderParams:
```

Comment:
Did we decide to use `@dataclass` + inheritance when declaring these frames? This syntax doesn't play well with the typing system

Reply:
yep missed one :)